be/src/exec/schema_scanner/schema_be_tablets_scanner.cpp

```diff
@@ -45,7 +45,7 @@ SchemaScanner::ColumnDesc SchemaBeTabletsScanner::_s_columns[] = {
         {"SCHEMA_HASH", TYPE_BIGINT, sizeof(int64_t), false},
         {"INDEX_DISK", TYPE_BIGINT, sizeof(int64_t), false},
         {"MEDIUM_TYPE", TYPE_VARCHAR, sizeof(StringValue), false},
-
+        {"NUM_SEGMENT", TYPE_BIGINT, sizeof(int64_t), false},
```

Comment:
This change should be testified compatibility in gray-scale update.

Reply:
@mofeiatwork Is there a compatibility issue in gray-scale update? it seems that you have encounter the same issue before.

Reply:
There is no compatibility problem for scanner get schema from fe